src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -46,6 +46,8 @@ internal sealed override CommandLineArguments CommonParse(IEnumerable<string> ar
         /// <returns>a commandlinearguments object representing the parsed command line.</returns>
         public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string baseDirectory, string sdkDirectory, string additionalReferenceDirectories = null)
         {
+            Debug.Assert(baseDirectory == null || PathUtilities.IsAbsolute(baseDirectory));
```

Comment:
`PathUtilities.IsAbsolute` returns false for a null

Reply:
Does VB need a similar assert?

---
In reply to: [223804356](https://github.com/dotnet/roslyn/pull/30383#discussion_r223804356) [](ancestors = 223804356)

Reply:
Yes, thanks. Will add